docs/components/components-overview.md

```diff
@@ -4,6 +4,7 @@ title: Overview Components
 sidebar_label: Overview Components
 slug: /components/
 description: "This section contains product manual content for each component in Camunda Platform 8, including conceptual content."
+keywords: ["process automation tools"]
```

Comment:
👍 

I had to think through the impact of putting the keywords in both the "next" version and 8.2. I agree with it, but here's my thought process on why I think you included it. Let me know if I got this wrong! 

It might seem like the keywords for `next` would fight SEO-wise with the keywords for 8.2, but they shouldn't, because `next` shouldn't be indexed by crawlers. (The entire `next` version is excluded by the sitemap, which should suggest to crawlers that they shouldn't be indexed). 

Putting them here in `next` is more about making sure they are present in version 8.3 pages when we cut that version.

Reply:
Yes 100%. I don't trust myself to add them in 8.2 and not remember to create another PR if this experiment is successful to add them to `next`. I'd rather add them in both places now and remove them in both places (if necessary).